### ISSUE TITLE
Transactions page for a wallet address doesn’t show minted NFTs

### ISSUE THREAD

**Issue by nhlroyrocks** [blockscout/blockscout#4581]
Is there a spot on this website where I can see all the NFTs that have been minted to my wallet address?

**Comment by vbaranov** [comment]
They should be listed in the tokens dropdown on the top right balance container at the address page of the target address: `/address/0x...`

<img width="649" alt="Screenshot 2021-08-30 at 16 41 19" src="https://user-images.githubusercontent.com/4341812/131348513-faae58cd-0e0a-46e9-bbbf-780de9aa4ce8.png">

Were you able to find it?

**Comment by nhlroyrocks** [comment]
Yes, the place I got hooked up is that a wallet address search results in the transactions page being shown by default and I didn’t notice the little drop down arrow to let me specify the /token page.

**Comment by vbaranov** [comment]
Thanks for the feedback. We are working on making the list of tokens more prominent. I am closing this for now.
